docs/html/_sources/release.rst.txt

```diff
@@ -0,0 +1,146 @@
+How to Prepare a Release
+========================
+
+In this guide, we'll cover:
+
+* preparing the code for a release
+* releasing the code on Github
+* releasing the software on Anaconda
+
+
+
+Preparing The Code For Release
+------------------------------
+
+These steps entail modifying files before a release is made.
+
+1. Pull the lastest code from whatever branch you want to release from.
+It's usually ``master``.
+
+    ::
+
+        git checkout master
+        git pull origin master
+
+Or checkout a branch:
+
+    ::
+
+        git fetch origin master
+        git checkout -b <branch-name> origin/master
+
+2. Edit ``version`` in ``setup.py`` to the new version.
+Don't prefix this with a "v".
+
+3. Edit ``__version__`` in ``zstash/__init__.py``. Prefix this with a "v".
+
+5. Change the ``version``  in ``conda/meta.yaml``.
+``version`` is what the version of the software will be on Anaconda.
+Don't prefix this with a "v".
+
+7. Commit and push your changes.
+
+    ::
+
+        git commit -am 'Update version numbers'
+        git push origin master
+
+Or:
+
+    ::
+
+        git commit -am 'Update version numbers'
+        git push <fork-name> <branch-name>
+        # Create pull request for the gh-pages branch
+
+Creating A Release On GitHub
+----------------------------
+
+1. Go to the Releases on the GitHub repo of the project
+`here <https://github.com/E3SM-Project/zstash/releases>`_.
+and draft a new release.
+
+2. ``Tag version`` and ``Release title`` should both be the version, including the "v".
+``Target`` should be ``master``. Use ``Describe this release`` to write what features
+the release adds. You can scroll through
+`Zstash commits <https://github.com/E3SM-Project/zstash/commits/master>`_ to see
+what features have been added recently.
+
```

Comment:
Perhaps here is where to add 3. on downloading the tar.gz file, getting the `sha256` checksum, and making a PR to update it in the `conda/meta.yaml`